packages/flutter_tools/lib/src/build_system/targets/macos.dart

```diff
@@ -159,6 +189,7 @@ class ReleaseUnpackMacOS extends UnpackMacOS {
   List<Source> get inputs => <Source>[
     ...super.inputs,
     const Source.artifact(Artifact.flutterMacOSXcframework, mode: BuildMode.release),
+    const Source.artifact(Artifact.flutterMacOSXcframework, mode: BuildMode.release),
```

Comment:
```suggestion
    const Source.artifact(Artifact.flutterMacOSFrameworkDsym, mode: BuildMode.release),
```

Reply:
Oh nice catch -- I fixed that one in a couple places, but definitely missed this one.

Reply:
Oh wait no this shouldn't be necessary. The dSYM is embedded within the XCFramework here.